src/cljs/org/broadinstitute/firecloud_ui/page/workspace/monitor/common.cljs

```diff
@@ -0,0 +1,33 @@
+(ns org.broadinstitute.firecloud-ui.page.workspace.monitor.common
+  (:require
+    [dmohs.react :as react]
+    [org.broadinstitute.firecloud-ui.common :as common]
+    [org.broadinstitute.firecloud-ui.common.components :as comps]
+    [org.broadinstitute.firecloud-ui.common.icons :as icons]
+    [org.broadinstitute.firecloud-ui.common.style :as style]
+    [org.broadinstitute.firecloud-ui.common.table :as table]
+    [org.broadinstitute.firecloud-ui.page.workspace.monitor.workflow-details :as workflow-details]
+    [org.broadinstitute.firecloud-ui.endpoints :as endpoints]
+    ))
+
+
+(defn render-date [date]
```

Comment:
Why is this here and not in utils? Or style? I'm not sure what the best home for it would be, but it seems much more generic than monitor.